apps/maestro/src/ui/pages/InterchainTokenDetailsPage/ConnectedInterchainTokensPage.tsx

```diff
@@ -228,7 +228,7 @@ const ConnectedInterchainTokensPage: FC<ConnectedInterchainTokensPageProps> = (
       sourceChainId: interchainToken?.chain?.id ?? "",
       destinationChainIds,
       gasLimit: 1_000_000,
-      gasMultipler: 3,
+      gasMultipler: 1.5,
```

Comment:
This is fine for now. But in the future, with the new sdk release, since we've already incorporated axelaron's `gasMultiplier` when it's undefined, can we just leave it undefined and use axelaron's `gasMultiplier`? since each chain should have different gasMultiplier value.